app/features/margin.js

```diff
@@ -50,16 +50,14 @@ export function pushElement(els, direction) {
           ? payload.current - payload.amount
           : payload.current + payload.amount
       }))
-    .forEach(({el, style, margin}) =>
-      el.style[style] = `${margin < 0 ? 0 : margin}px`)
+    .forEach(({el, style, margin}) => el.style[style] = `${margin}px`)
```

Comment:
intentional to enable negative margins? that's all this line change does.

Reply:
Yes. That's the intention.